packages/server-side-render/src/server-side-render.js

```diff
@@ -88,20 +87,12 @@ export default function ServerSideRender( props ) {
 
 		setIsLoading( true );
 
-		const sanitizedAttributes =
-			attributes &&
-			__experimentalSanitizeBlockAttributes( block, attributes );
```

Comment:
Why were we previously sanitising the attributes here?

Reply:
Previously we needed this because the endpoint called by this component validates attributes, which would break with the previous implementation of `__internalWidgetId` (since it wasn't a registered attribute).

With the new implementation, `__internalWidgetId` is a perfectly valid attribute and it's no longer necessary to sanitize here.

Reply:
Although after rebasing I'm actually getting those validation errors again when I test an SSR block in the widget editor -- so maybe I spoke too soon! Taking a look to see what's changed, as I would still expect this to work without sanitization.